assets/wizards/subscriptions/views/manageSubscriptionsScreen.js

```diff
@@ -71,31 +62,9 @@ class ManageSubscriptionsScreen extends Component {
 						checked={ choosePrice }
 					/>
 				) }
-				<Button
-					isPrimary
-					className="is-centered"
-					onClick={ () =>
-						onClickEditSubscription( {
-							id: 0,
-							name: '',
-							image: null,
-							price: '',
-							frequency: 'month',
-						} )
-					}
-				>
-					{ buttonText }
-				</Button>
-				<Button
-					className='is-centered'
-					isTertiary
-					href={ newspack_urls['checklists']['memberships'] }
-				>
-					{ __( "I'm done adding" ) }
-				</Button>
 			</div>
 		);
 	}
 }
 
-export default ManageSubscriptionsScreen;
+export default withWizardScreen( ManageSubscriptionsScreen );
```

Comment:
This should have the `noBackground` prop on `withWizardScreen`, otherwise there are white cards on a white background:

![localhost_wp-admin_admin php_page=newspack-subscriptions-wizard](https://user-images.githubusercontent.com/7317227/60045683-4cca1780-967a-11e9-9d52-892cf68936a8.png)

Reply:
Fixed in https://github.com/Automattic/newspack-plugin/pull/109/commits/ae3d9296287c1f5d42557bdece9c4329d4264444